src/screens/SendFunds/SelectFunds/index.js

```diff
@@ -0,0 +1,218 @@
+/* @flow */
+import React, { Component } from "react";
+import {
+  SafeAreaView,
+  View,
+  StyleSheet,
+  KeyboardAvoidingView,
+} from "react-native";
+import { connect } from "react-redux";
+import { BigNumber } from "bignumber.js";
+import { valueFromUnit } from "@ledgerhq/live-common/lib/helpers/currencies/valueFromUnit";
+import type { NavigationScreenProp } from "react-navigation";
+import type { Account } from "@ledgerhq/live-common/lib/types";
+import type { BigNumber as BigNumberType } from "bignumber.js";
+
+import { accountScreenSelector } from "../../../reducers/accounts";
+
+import colors from "../../../colors";
+
+import LText from "./../../../components/LText/index";
+import CurrencyUnitValue from "../../../components/CurrencyUnitValue";
+import CounterValue from "../../../components/CounterValue";
+import OutlineButton from "../../../components/OutlineButton";
+import BlueButton from "../../../components/BlueButton";
+
+import AmountInput from "./AmountInput";
+import CounterValuesSeparator from "./CounterValuesSeparator";
+
+type Props = {
+  account: Account,
+  navigation: NavigationScreenProp<{
+    accountId: string,
+    address: string,
+  }>,
+};
+
+type State = {
+  amount: string,
+  amountBigNumber: BigNumberType,
+};
+
+class SelectFunds extends Component<Props, State> {
+  static navigationOptions = {
+    title: "Amount",
+  };
+
+  state = {
+    amount: "0",
+    amountBigNumber: new BigNumber(0),
+  };
+
+  isAmountNaN = (amount: string): boolean => isNaN(parseFloat(amount));
+
+  onChangeText = (amount: string) => {
+    if (!this.isAmountNaN(amount)) {
+      const { account } = this.props;
+      const num = new BigNumber(parseFloat(amount));
+      const big = valueFromUnit(num, account.unit);
+      this.setState({
+        amount,
+        amountBigNumber: big,
+      });
+    }
+  };
+
+  navigate = () => {
+    const {
+      account,
+      navigation: {
+        state: {
+          // $FlowFixMe
+          params: { address },
+        },
+      },
+    } = this.props;
+    const { amount, amountBigNumber } = this.state;
+    this.props.navigation.navigate("SendSummary", {
+      accountId: account.id,
+      address,
+      amount,
+      amountBigNumber,
+    });
+  };
+
+  render() {
+    const { account } = this.props;
+    const { amount, amountBigNumber } = this.state;
+
+    const isWithinBalance = amountBigNumber.lt(account.balance);
+    const isValid = amountBigNumber.gt(0) && isWithinBalance;
```

Comment:
Notes for later: we might not be able to do it this way, we will have to ask the "bridge" / the libcore to know if the amount is below balance, because it involve fees, etc..